app/Http/Kernel.php

```diff
@@ -33,8 +33,9 @@ class Kernel extends HttpKernel
             \Coyote\Http\Middleware\FirewallBlacklist::class
         ],
         'api' => [
-            'throttle:60,1',
-            'bindings'
+            \Illuminate\Session\Middleware\StartSession::class,
+            \Illuminate\Routing\Middleware\SubstituteBindings::class,
+            Middleware\DefaultBindings::class,
```

Comment:
Ok, spojrzałem na kod i już wiem o co chodzi i czemu nie działa bez `StartSession`. Nieprzypadkowo middleware `StartSession` jest tylko w grupie `web`. 

Pisząc API nie używamy sesji, ponieważ API nie jest w stanie zachować sesji pomiędzy requestami.  Więcej informacji tutaj: https://laravel.com/docs/5.4/passport#introduction

Jeżeli więc potrzebujesz wygenerować token (na potrzeby chatu) dla aktualnie zalogowanego użytkownika (zalogowanego na podstawie cookies) to może faktycznie - Twój pierwotny kod i pomysł był ok. Niepotrzebnie trułem Ci o subdomenie `api` o osobnych middleware itp itd.

Jeżeli będziemy pisać prawdziwe API dla 4programmers.net, to możemy to zrobić na osobnej subdomenie, a autoryzacją poprzez token, oauth itp itd.

Ty potrzebujesz tego właściwie tylko do chatu tak więc sorry za zamieszanie...